### ISSUE TITLE
New ML Idea: learning rate schedules for gradient boosting trees

### ISSUE THREAD

**Issue by ClimbsRocks** [ClimbsRocks/auto_ml#263]
learning rate schedules work really well for neural nets. might they work well for other types of sequentially-updated problems as well?

this one might be a bit tricker to implement. it might require us to build our own gbdt implementation.
